app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.pintraveler.ptkit">
+
+    <application
+        android:allowBackup="true"
+        android:dataExtractionRules="@xml/data_extraction_rules"
+        android:fullBackupContent="@xml/backup_rules"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.PtKit"
+        tools:targetApi="31">
+        <activity
+            android:name="com.pintraveler.ptkit.MainActivity"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
```

Comment:
@eneseray bu bir application olmamasina ragmen manifestte <application> gerekiyor mu gercekten?

Reply:
internette birçok örnekte bu şekilde yapılıyordu. sanırım daha öncesinde böyle olmadığı için jitpack de hata alıyorduk